src/AuditTrailModule/javasource/test_crm/tests/expected/ExpectedLog.java

```diff
@@ -40,7 +57,28 @@ public void assertEqual(final Log log) throws CoreException {
 		assertTrue("Time of change before test was run.", log.getDateTime().after(this.afterDate));
 		assertTrue("Time of change after this assertion.", log.getDateTime().before(new Date()));
 
-		assertNull("There should be no reference to a superclass for this log.", log.getLog_AudittrailSuperClass());
+		if (this.superClassName == null) {
+			assertNull("There should be no reference to a superclass for this log.", log.getLog_AudittrailSuperClass());
+		} else {
+			assertEquals("Incorrect superclass for log.", this.superClassName, log.getLog_AudittrailSuperClass().getMendixObject().getType());
+		}
+
+		// Now verify loglines:
+		assertEquals("Incorrect number of loglines.", this.logLines.size(), actualLog.getLogLines().size());
+		final Set<String> visitedLogLines = new HashSet<String>();
+		for (final ActualLogLine actualLogLine : actualLog.getLogLines()) {
+			final String memberName = actualLogLine.getLogLineMemberName();
+
+			// Have we seen this one before?
+			assertFalse("Repeated log line for member " + memberName, visitedLogLines.contains(memberName));
+			visitedLogLines.add(memberName);
+
+			// Should we?
+			assertTrue("Expected logline not found for the member named " + memberName, this.logLines.containsKey(memberName));
```

Comment:
Nitpick - There can be a space between log and line.

Reply:
Or written in camel case, since it's an entity name.